### PR TITLE
Add Thai translations

### PR DESCRIPTION
**Pull request by ellipszist**

### CODE REVIEW

File: CJBCheatsMenu/i18n/th.json [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+    "tabs.time": "เวลา",
+    "tabs.advanced": "ตั้งค่าขั้นสูง",
+    "tabs.controls": "ตัวควบคุม",
+
+
+    /*********
+    ** Player & Tools tab
+    *********/
+    // player
+    "player.title": "ĕÇÅÐ¤Ãڌà˕¹", // ตัวละครผู้เล่น
```
Comment (Pathoschild) [inline CJBCheatsMenu/i18n/th.json:24]: Some of these translations seem to have corrupted text?

File: CJBCheatsMenu/i18n/th.json [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    "skills.increase-farming": "เพิ่มค่าทักษะ การทำฟาร์ม : {{currentLevel}}",
+    "skills.increase-mining": "เพิ่มค่าทักษะ การทำเหมือง: {{currentLevel}}",
+    "skills.increase-foraging": "เพิ่มค่าทักษะ การหาของป่า: {{currentLevel}}",
+    "skills.increase-fishing": "เพิ่มค่าทักษะ การตกปลา: {{currentLevel}}",
+    "skills.increase-combat": "เพิ่มค่าทักษะ การต่อสู้: {{currentLevel}}",
+    "skills.reset": "รีเซ็ต ค่าทักษะ!",
+
+    // professions
+    "professions.title": "ÍÒƚ¾", // อาชีพ
+    "professions.combat.fighter": "Combat lvl 5 - Fighter",
```
Comment (Pathoschild) [inline CJBCheatsMenu/i18n/th.json:89]: These seem to be untranslated?

File: CJBItemSpawner/i18n/th.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+    /*********
+    ** Sort button
+    *********/
+    "sort.by-name": "ชื่อ",
+    "sort.by-type": "ประเภท",
+    "sort.by-id": "ID",
```
Comment (Pathoschild) [inline CJBItemSpawner/i18n/th.json:7]: This should probably be translated too. (ID means the internal item identifier.)

File: CJBCheatsMenu/i18n/th.json (hunk 1 above)
Comment (ellipszist) [reply CJBCheatsMenu/i18n/th.json:24]: @Pathoschild We matched Thai characters with [unicode tables](https://unicode-table.com/en/) to create **LooseSprites/font_bold.png** files. which works well with stardew.



Ref: https://github.com/ellipszist/StardewMods/tree/master/_encodes/StardewValley/assets/Content/LooseSprites

File: CJBItemSpawner/i18n/th.json (hunk 3 above)
Comment (ellipszist) [reply CJBItemSpawner/i18n/th.json:7]: @Pathoschild ID or some simple English word and generally used, Thai people can understand better than to translate into Thai

**Review by Pathoschild — APPROVED** [review]

**Comment by Pathoschild** [comment]
Thanks for the help! Merged into `develop` for the upcoming CJB Cheats Menu 1.29.3, CJB Item Spawner 2.1.5, and CJB Show Item Sell Price 1.9.13.

**Pull request merged**